source/code/plugins/in_auoms.rb

```diff
@@ -0,0 +1,151 @@
+#
+# Fluentd
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+require 'fileutils'
+require 'socket'
+
+require 'cool.io'
+require 'yajl'
+
+require 'fluent/input'
+require 'fluent/event'
+
+module Fluent
+  # obsolete
+  class AuOMSInput < Input
+    Plugin.register_input('auoms', self)
+
+    config_param :blocking_timeout, :time, default: 0.5
+
+    desc 'Tag of the output events.'
+    config_param :tag, :string, default: nil
+    desc 'The path to your Unix Domain Socket.'
+    config_param :path, :string, default: nil
+    desc 'The backlog of Unix Domain Socket.'
+    config_param :backlog, :integer, default: nil
+
+    def initialize
+      require 'socket'
+      require 'yajl'
```

Comment:
You already require yajl and socket at the top of the file; why do you need them here?